### PR TITLE
Add metrics description for serverless tier

### PR DESCRIPTION
**Pull request by sunxiaoguang**
Signed-off-by: Xiaoguang Sun <sunxiaoguang@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/monitor-tidb-cluster.md [hunk 1]
```diff
@@ -65,6 +65,13 @@ The cluster overview page provides general metrics of a cluster, including total
 >
 > Some of these metrics might be available only for Dedicated Tier clusters.
 
+| Serverless Tier metric | Description |
+|:--|:--|
+| **Total QPS** | The number of SQL statements executed per second. |
+| **Average Query Duration** | The average of execution time for SQL statements. |
+| **Active Connections** | Number of connections that are actively executing SQL statements. |
```
Comment (ran-huang) [inline tidb-cloud/monitor-tidb-cluster.md:72]: To keep them consistent with the console
```suggestion
| **QPS** | The number of SQL statements executed per second. |
| **Query Duration** | The average execution time for SQL statements. |
| **Active Connection** | Number of connections that are actively executing SQL statements. |
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: tidb-cloud/monitor-tidb-cluster.md [hunk 2]
```diff
@@ -65,6 +65,13 @@ The cluster overview page provides general metrics of a cluster, including total
 >
 > Some of these metrics might be available only for Dedicated Tier clusters.
 
+| Serverless Tier metric | Description |
```
Comment (ran-huang) [inline tidb-cloud/monitor-tidb-cluster.md:68]: duplicate with https://github.com/pingcap/docs/pull/11654?

**Comment by sunxiaoguang** [comment]
Duplicated by #11654, close this PR.

**Pull request closed without merging**